src/flows/pipes/Notification/view.tsx

```diff
@@ -675,4 +716,10 @@ ${ENDPOINT_DEFINITIONS[data.endpoint]?.generateTestQuery(data.endpointData)}`
   )
 }
 
-export default Notification
+const mapStateToProps = state => {
+  return {
+    secrets: getAllSecrets(state),
+  }
+}
+
+export default connect(mapStateToProps, null)(Notification)
```

Comment:
why not use a hook for these? This component already uses redux hooks

Reply:
a fair point. Let me refactor